.github/workflows/test.yml

```diff
@@ -0,0 +1,18 @@
+name: Tests
+on: [push]
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go 1.13
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13.3
```

Comment:
could we also add a separate stage to test if the code works with 1.12 version?

Reply:
Added [build matrix](https://help.github.com/en/github/automating-your-workflow-with-github-actions/configuring-a-workflow#configuring-a-build-matrix) that runs test on multiple versions of golang. See the result: https://github.com/kiwicom/easycql/runs/281504032